### ISSUE TITLE
Mismatch of parameters of loss function of CategoricalCrossentropy 

### ISSUE THREAD

**Issue by glcLucky** [danielegrattarola/spektral#78]
Hi, thanks for your great project. That's very useful for me. 
But, when I ran the script of BDGC_disjoint.py, I found there was a big difference in the loss value between the training set and the validation set while the accuracy was almost the same, which was not reasonable. I carefully checked the code of calculating loss. Inside the function of evaluate,  there is such a line: outs = [o(pred, y) for o in ops_list], which aim to calculate the loss and accuracy. You use the loss function of CategoricalCrossentropy, which requires to pass y and pred in turn, not pred and y(in your code). When I fixed the issue, the differenct between training set and validation set returned to normal. I'm not sure if my debug is right.Could you hele me confirm the issue? Thanks.

**Comment by danielegrattarola** [comment]
Thanks for spotting this, I have fixed it. 

Cheers
